verifier/helpers.go

```diff
@@ -60,10 +60,10 @@ func loadX509TrustStores(ctx context.Context, scheme signature.SigningScheme, po
 	return loadX509TrustStoresWithType(ctx, typeToLoad, policyName, trustStores, x509TrustStore)
 }
 
-// isCriticalFailure checks whether a VerificationResult fails the entire
-// signature verification workflow.
+// isCriticalFailure checks whether a [notation.ValidationResult] fails the
```

Comment:
I'm not sure if cross package linking works or not. Could you verify it?

Reply:
Yep, it works. Here's the official doc: https://go.dev/doc/comment#links, see `Doc links`. @shizhMSFT